### PR TITLE
[FLINK-11610][docs-zh] Translate the "Examples" page into Chinese

### PR DESCRIPTION
**Pull request by LakeShen**
## What is the purpose of the change

*Translate the "Examples" page into Chinese*

## Brief change log

*(for example:)*
  - *create a new file named  index.zh.md,which located in flink/docs/examples/index.zh.md*

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by wuchong** [comment]
Could rebase to master and fix the conflicts first?

**Comment by YueYeShen** [comment]
> Could you rebase to master and fix the conflicts first?

ok,I have fixed the conflicts.

File: docs/examples/index.zh.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
 title: 示例
 nav-id: examples
-nav-title: '<i class="fa fa-file-code-o title appetizer" aria-hidden="true"></i> 示例'
+
+nav-title: '<i class="fa fa-file-code-o title appetizer" aria-hidden="true"></i> Examples'
```
Comment (wuchong) [inline docs/examples/index.zh.md]: Revert this?

Comment (YueYeShen) [reply docs/examples/index.zh.md]: ok,thank you ,I will do that.

File: docs/examples/index.zh.md [hunk 2]
```diff
@@ -26,25 +26,23 @@ under the License.
...
-## Bundled Examples
+## 附带示例
 
-The Flink sources include many examples for Flink's different APIs:
+在Flink源文件中，包含了许多Flink不同API接口的代码示例：
 
-* DataStream applications ({% gh_link flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples "Java" %} / {% gh_link flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples "Scala" %}) 
-* DataSet applications ({% gh_link flink-examples/flink-examples-batch/src/main/java/org/apache/flink/examples/java "Java" %} / {% gh_link flink-examples/flink-examples-batch/src/main/scala/org/apache/flink/examples/scala "Scala" %})
-* Table API / SQL queries ({% gh_link flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java "Java" %} / {% gh_link flink-examples/flink-examples-table/src/main/scala/org/apache/flink/table/examples/scala "Scala" %})
+* DataStream 应用 ({% gh_link flink-examples/zh/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples "Java" %} / {% gh_link flink-examples/zh/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples "Scala" %})
```
Comment (wuchong) [inline docs/examples/index.zh.md]:
```suggestion
* DataStream 应用 ({% gh_link flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples "Java" %} / {% gh_link flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples "Scala" %}) 
```

File: docs/examples/index.zh.md [hunk 3]
```diff
@@ -26,25 +26,23 @@ under the License.
...
-* DataSet applications ({% gh_link flink-examples/flink-examples-batch/src/main/java/org/apache/flink/examples/java "Java" %} / {% gh_link flink-examples/flink-examples-batch/src/main/scala/org/apache/flink/examples/scala "Scala" %})
-* Table API / SQL queries ({% gh_link flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java "Java" %} / {% gh_link flink-examples/flink-examples-table/src/main/scala/org/apache/flink/table/examples/scala "Scala" %})
+* DataStream 应用 ({% gh_link flink-examples/zh/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples "Java" %} / {% gh_link flink-examples/zh/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples "Scala" %}) 
+* DataSet 应用 ({% gh_link flink-examples/zh/flink-examples-batch/src/main/java/org/apache/flink/examples/java "Java" %} / {% gh_link flink-examples/zh/flink-examples-batch/src/main/scala/org/apache/flink/examples/scala "Scala" %})
```
Comment (wuchong) [inline docs/examples/index.zh.md]:
```suggestion
* DataSet 应用 ({% gh_link flink-examples/flink-examples-batch/src/main/java/org/apache/flink/examples/java "Java" %} / {% gh_link flink-examples/flink-examples-batch/src/main/scala/org/apache/flink/examples/scala "Scala" %})
```

File: docs/examples/index.zh.md [hunk 4]
```diff
@@ -26,25 +26,23 @@ under the License.
...
+* Table API / SQL 查询 ({% gh_link flink-examples/zh/flink-examples-table/src/main/java/org/apache/flink/table/examples/java "Java" %} / {% gh_link flink-examples/zh/flink-examples-table/src/main/scala/org/apache/flink/table/examples/scala "Scala" %})
```
Comment (wuchong) [inline docs/examples/index.zh.md]:
```suggestion
* Table API / SQL 查询 ({% gh_link flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java "Java" %} / {% gh_link flink-examples/flink-examples-table/src/main/scala/org/apache/flink/table/examples/scala "Scala" %})
```

File: docs/examples/index.zh.md (hunk 2 above)
Comment (wuchong) [reply docs/examples/index.zh.md]: The code link doesn't provide Chinese version.

**Comment by wuchong** [comment]
Thanks @YueYeShen , the translation looks good to me now. 

Will merge it.

**Comment by YueYeShen** [comment]
> Thanks @YueYeShen , the translation looks good to me now.
> 
> Will merge it.

Thank you , Jark

**Pull request closed without merging**